src/nunit-gui/Presenters/TestPropertiesPresenter.cs

```diff
@@ -185,5 +202,29 @@ private string TrimMessage(string message)
         }
 
         #endregion
+
+        #region Nested AssertionResult Class
```

Comment:
I feel like this file is getting too large anyway, and now with a region added with a class in it just kind of rubs me the wrong way. I'm assuming it's here with the thought of when this all gets separated it will be easier... but it still sort of bugs me.